MVC/Views/Home/NullPost.cshtml

```diff
@@ -1,5 +1,5 @@
-@model string
-<h1>Post Not Found</h1>
-<h5>@Model</h5>
+@model string[]
+<h1>@Model[0]</h1>
+<h5>@Model[1]</h5>
```

Comment:
This assumes the model will have two strings. We should find a way to break these assumptions.